components/board/genericlinux/board_test.go

```diff
@@ -164,24 +131,6 @@ func TestGenericLinux(t *testing.T) {
 		test.That(t, err.Error(), test.ShouldContainSubstring, "closed")
 		test.That(t, rx, test.ShouldBeNil)
 	})
-
-	t.Run("test software pwm loop", func(t *testing.T) {
-		newCtx, cancel := context.WithTimeout(ctx, time.Duration(10))
-		defer cancel()
-		gp2.b.softwarePWMLoop(newCtx, *gp2)
```

Comment:
I would rather test our software pwm loop directly if we can no longer access it through these tests. Is it just perhaps that you very eagerly removed and `gp.b` object? These functions should all still be part of the genericlinux package as far as I recall.

Reply:
Unreoslving this to track the tests that will be added as discussed.